horizon/enforcer/api.py

```diff
@@ -162,20 +152,18 @@ def log_query_result_kong(input: KongAuthorizationInput, response: Response):
         format = color + "is allowed = {allowed} </>"
         format += " | <cyan>{api_params}</>"
         if sidecar_config.DECISION_LOG_DEBUG_INFO:
-            format += (
-                " | full_input=<fg #fff980>{input}</> | debug=<fg #f7e0c1>{debug}</>"
-            )
+            format += " | full_input=<fg #fff980>{input}</> | debug=<fg #f7e0c1>{debug}</>"
         logger.opt(colors=True).info(
             format,
             allowed=allowed,
             api_params=params,
             input=input.dict(),
             debug=debug,
         )
-    except:
+    except Exception:  # noqa: BLE001
         try:
             body = str(response.body, "utf-8")
-        except:
+        except ValueError:
```

Comment:
make sure this is the only exception we catch here

Reply:
It's only parsing bytes to string, what else would you expect?

Reply:
didn't expect anything, just commented to verify